Lib/dataclasses.py

```diff
@@ -532,6 +537,79 @@ def _hash_fn(fields):
                       [f'return hash({self_tuple})'])
 
 
+def _is_classvar(a_type, typing):
+    if typing:
+        # This test uses a typing internal class, but it's the best
+        #  way to test if this is a ClassVar.
```

Comment:
Remove one extra space before "way to ..." (there are two instead of one).

Reply:
I like my extra spaces to continue the same paragraph. But I agree they should probably go away. I'll do that systematically in a separate checkout.